app/controllers/reasons/UnderpaymentReasonAmendmentController.scala

```diff
@@ -101,14 +101,14 @@ class UnderpaymentReasonAmendmentController @Inject() (
     boxNumber match {
       case BoxNumber.Box22 | BoxNumber.Box37 | BoxNumber.Box39 | BoxNumber.Box41 | BoxNumber.Box42 | BoxNumber.Box62 |
           BoxNumber.Box63 | BoxNumber.Box66 | BoxNumber.Box67 | BoxNumber.Box68 =>
-        textAmendmentView(form, formAction(boxNumber), boxNumber, itemNumber, backLink(boxNumber))
+        textAmendmentView(form, formAction(boxNumber), boxNumber, itemNumber, None)
       case BoxNumber.Box33 =>
         textAmendmentView(
           form,
           formAction(boxNumber),
           boxNumber,
           itemNumber,
-          backLink(boxNumber),
+          None,
```

Comment:
do we still want the view to take in this as an optional if we're only setting it to none everywhere? could just take it out and set to none in the view